docs/internals/vsr.md

```diff
@@ -271,9 +271,9 @@ Sync missed client replies using [Protocol: Repair Grid](#protocol-repair-client
 (Runs immediately after [Protocol: Sync Superblock](#protocol-sync-superblock).)
 See [State Sync](./sync.md) for details.
 
-## Protocol: Sync Tables
+## Protocol: Sync Forest
 
-Sync missed LSM table blocks using [Protocol: Repair Grid](#protocol-repair-grid).
+Sync missed LSM manifest and table blocks using [Protocol: Repair Grid](#protocol-repair-grid).
```

Comment:
This is sort of aspirational atm. Right now manifest blocks are still repaired via regular grid repair -- i.e., the syncing replica reads the block, sees that it's missing, and repairs that block. But that limits manifest repair to be sequential.

Long-term:
- the healthy replica should see notice when a manifest block is requested, and send long with it the handful of manifest blocks that precede it
- the syncing replica should cache blocks that it receives before it has requested, just in case it needs them momentarily.